pipelines/testing/ImputationBeagleEmpty.wdl

```diff
@@ -0,0 +1,53 @@
+version 1.0
+
+# name it ImputationBeagle even though it's empty for testing
+workflow ImputationBeagle {
+
+    String pipeline_version = "0.0.1"
+
+    input {
+        Int chunkLength = 25000000
+        Int chunkOverlaps = 5000000 # this is the padding that will be added to the beginning and end of each chunk to reduce edge effects
```

Comment:
should we move all the stuff that isnt a server/user input out of the input block in this wdl and the real wdl or wait to do that when we're more sure of what these will be (after Kyle reviews it)?